.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile

```diff
@@ -155,7 +155,29 @@ pipeline {
                             runWith(lib, failedTests, "eck-84-${BUILD_NUMBER}-e2e", "8.4.3")
                         }
                     }
-               }
+                }
+                stage("8.5.x") {
+                    agent {
+                        label 'linux'
+                    }
+                    steps {
+                        unstash "source"
+                        script {
+                            runWith(lib, failedTests, "eck-85-${BUILD_NUMBER}-e2e", "8.5.3")
+                        }
+                    }
+                }
+                stage("8.6.x") {
```

Comment:
Do we need 8.6 here if it is the default version for all other test pipelines?

Reply:
Indeed, we don't need it.